components/compliance-service/habitat/default.toml

```diff
@@ -22,4 +22,4 @@ manual_polling_interval = 1440
 [agent]
 workers = 10
 buffer_size = 1000
-remote_inspec_version = "REPLACE-FROM-PLAN.SH"
+remote_inspec_version = "4.7.24"
```

Comment:
:x: I don't think this should be committed. I do think I have commented this somewhere else already 😉 Probably the PR this is based on... 🤔

Reply:
this one is annoying
`git checkout origin/master -- components/compliance-service/habitat/default.toml` is in my history, don't know when i rebuilt the thing